Wiki/weight-tracker.md

```diff
@@ -1,30 +1,25 @@
-# Weight-tracker
-Weight Tracker is a [Script API](script-api.md) showcase present in the [demo document](document.md).
+# Weight-tracker
 
-[Day notes](day-notes.md) shows (among others) how we have "weight" [promoted attribute](promoted-attributes.md) in the day note [template](template.md). This then aggregates the data and shows a nice chart of weight change in time.
 
-Demo
-----
+The `Weight Tracker` is a [Script API](script-api.md) showcase present in the [demo document](document.md).
 
-![](images/weight-tracker.png)
+By adding `weight` as a [promoted attribute](promoted-attributes.md) to the [Day notes](day-notes.md) through its [template](template.md), you can aggregate the data and plot weight change over time.
 
-How to remove Weight Tracker button from the top bar
-----------------------------------------------------
+![](images/weight-tracker.png)
 
-In the link map of Weight Tracker, there is a note "Button". Open it and delete or comment out its contents. Weight Tracker button will disappear after you close and open the app.
+## Implementation
 
-Implementation
---------------
+The `Weight Tracker` Note in the screenshot above is of type `Render Note`. That type of note doesn't have any useful content itself, rahter it provides a place where a [script](scripts.md) can render its output.
```

Comment:
`Weight Tracker` **_n_**ote
is of **_the_** type `Render Note`
rahter > rather  >> or perhaps more clear, break sentence:
 [...] itself. Instead, it is a placeholder where a [script](scripts.md) can render its output.